01week/rockPaperScissors.js

```diff
@@ -12,6 +12,22 @@ function rockPaperScissors(hand1, hand2) {
 
   // Write code here
 
+  
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
+
+  
+
+  if(hand1 === hand2){
+    return "It's a tie!"
+  }
+  if(hand1 === 'rock' && hand2 === 'scissors'|| hand1 === 'scissors' && hand2 === 'paper' || hand1 === 'paper' && hand2 === 'rock'){
+    return 'hand1 wins!'
+  } else if 
+    
+  (hand2 === 'rock' && hand1 === 'scissors'|| hand2 === 'scissors' && hand1 === 'paper' || hand2 === 'paper' && hand1 === 'rock')
```

Comment:
Good job in building RPS; I like the order of operations you utilized with your logic.
Try to refactor this code so that you don't repeat yourself.